packages/actionmenu/package.json

```diff
@@ -19,6 +19,11 @@
     "test:updateSnapshot": "npm run test -- --updateSnapshot",
     "test:watch": "npm run test -- --watchAll"
   },
+  "jest": {
```

Comment:
Ugh, is this required on every package to get ride of requestAnimationFrame warning?  Do you know the origin of that warning at this point?